salt/tests/unit/modules/files/test_metalk8s_package_manager_yum.yaml

```diff
@@ -0,0 +1,147 @@
+list_dependents:
```

Comment:
You just have to read the yaml file to see all the test cases. If we put all in the python it will be unreadable to me.
We could add labels for tests  it's true

Reply:
I get why we don't want to clutter code files with many repetitive datasets. Just not fond of having things requiring my brain to split attention between multiple files when trying to reason about a piece of logic.

Let's keep this approach for now, better have tests than debate for too long, we'll see how that works in the long run.